README.md

```diff
@@ -10,6 +10,30 @@ specification.
 Actual plotting code is done by renderers that are provided by other plotting libraries.
 For the purpose or prototyping, we are shipping both a Matplotlib renderer and a simple embeddable HTML renderer in Altair.
 
+# Motivation
+
+There are currently many excellent plotting libraries in Python. The main ones are:
+
+* Matplotlib
+* Bokeh
+* Seaborn
+* Lightening
+* Plotly
+* Pandas built-in plotting
```

Comment:
Maybe mention holoviews as well, because it tries to do some of the auto-plot-inference we have in mind here